### ISSUE TITLE
BinderNode is auto imported from wrong location in VS Code

### ISSUE THREAD

**Issue by Artur-** [vaadin/fusion#20]
If you in a TS file type
```
BinderNode<cmd-space>
```

the suggestion is
![image](https://user-images.githubusercontent.com/260340/119235725-a116f400-bb3c-11eb-861a-6fc63a20a0f9.png)

When using the suggestion, you end up with
```
import { BinderNode } from 'Frontend/../target/flow-frontend/form/BinderNode';
```

The correct import would be
```
import { BinderNode } from '@vaadin/form/BinderNode';
```

**Comment by Artur-** [comment]
Seems `EndpointError` has a similar problem and ends up as
```
import { EndpointError } from 'Frontend/../target/flow-frontend/Connect';
```

when it should be
```
import { EndpointError } from '@vaadin/flow-frontend/Connect';
```

That is also a strange import path but seems like the correct one based on where the file is

**Comment by platosha** [comment]
The `EndpointError` is now imported from `@vaadin/fusion-frontend` after we introduced the package.

The `BinderNode` hopefully will be fixed by #21
